packages/slinkity/cli/index.js

```diff
@@ -55,7 +55,8 @@ const slinkityArgs = {
   },
   port: {
     flag: '--port <port>',
-    description: 'Port for Vite server (default: 3000)',
+    description: 'Port for Vite server (default: 8080)',
```

Comment:
```suggestion
    description: 'Port for Vite server',
```

Adding `(default: 8080)` will actually make the default print _twice_ in the `--help` page! Adding a `defaultValue` below generates this text already. I'd suggest removing the `(default...)` text from here and leaving that `defaultValue`